imports/plugins/core/orders/server/no-meteor/mutations/createRefund.js

```diff
@@ -0,0 +1,114 @@
+import SimpleSchema from "simpl-schema";
+import accounting from "accounting-js";
+import Logger from "@reactioncommerce/logger";
+import ReactionError from "@reactioncommerce/reaction-error";
+import { getPaymentMethodConfigByName } from "/imports/plugins/core/payments/server/no-meteor/registration";
+import sendOrderEmail from "../util/sendOrderEmail";
+
+const inputSchema = new SimpleSchema({
+  amount: {
+    type: Number,
+    min: 0.01
```

Comment:
Talked with Eric D about this. Thinking `min: 0, exclusiveMin: true` is probably safer since some currencies might have small fractional positive values (Bitcoin!).